Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/pending/SubmissionTestResultPendingFragment.kt

```diff
@@ -33,27 +33,20 @@ class SubmissionTestResultPendingFragment : Fragment(R.layout.fragment_submissio
 
     private var skipInitialTestResultRefresh = false
 
+    private var errorDialog: AlertDialog? = null
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        pendingViewModel.consentGiven.observe2(this) {
+        pendingViewModel.consentGiven.observe(viewLifecycleOwner) {
```

Comment:
Why did you change this?

Reply:
I do not understand why we use a wrapper here. It's just confusing to see for everyone new to the project.

Reply:
* To have shorter code, initially it removed the need for `(object : Observer...` which is no longer needed since newer androidx versions, but `this` is still shorter than `viewLifecycleOwner`.
* To reduce the chance someone writing `observe(this)`  and using the wrong lifecycle owner.

If it were possible I'd like to actually make it just `observe()` in fragments but I don't think Kotlin allows for that.

Reply:
reverted